src/bindings/python/src/openvino/frontend/frontend.py

```diff
@@ -15,6 +15,8 @@ def __init__(self, fe: FrontEndBase) -> None:
         super().__init__(fe)
 
     def convert(self, model: Union[Model, InputModel]) -> Model:
+        if isinstance(model, Model):
+            model = model._Model__model  # or change convert(model) to accept ie_api.Model
```

Comment:
Is this comment supposed to be in the final PR?

Reply:
Nah ;) it was easier to fix it here
`__model` should be internal and used only in pyopenvino and other "_" files. Here I believe I can move the changes to `pyopenvino/frontend/frontend.cpp`. But I am not sure about `test_utils\test_api.py`. Since I added this, maybe we can mark it as internal? 

About `openvino/runtime/ie_api.py`, there is a disscusion to rename it to `_ov_api` after `runtime` deprecation

Reply:
```sh
install(DIRECTORY ${OpenVINOPython_SOURCE_DIR}/src/openvino/test_utils
        DESTINATION tests/${OV_CPACK_PYTHONDIR}/openvino
        COMPONENT tests
        EXCLUDE_FROM_ALL)
```

Reply:
> But I am not sure about test_utils\test_api.py. Since I added this, maybe we can mark it as internal?

I suppose marking `test_api.py` as internal wouldn't hurt, I'd opt for doing it in a separate PR though.